cong-framework/python/core/__init__.py

```
@@ -0,0 +1,20 @@
+
+from .exp import Exp, eval, is_exp
+from .val import Val
+from .bind import Bind
+from .primitive import Number, String, Boolean, Real
+from .proj import Proj
+from .base import Base
```

Ruff (0.8.2)

`.base.Base` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)